airbyte-integrations/bases/base-python/setup.py

```diff
@@ -37,7 +37,7 @@
         "PyYAML==5.4",
         "pydantic==1.6.*",
         "airbyte-protocol",
-        "jsonschema==2.6.0",
+        "jsonschema==3.2.0",
```

Comment:
Are you sure that `jsonschema `version 3 is backward compatible with version 2 and this change will not generate new bugs?

Reply:
not 100% sure but acceptance test passed fine. There were dependencies conflicts with SAT that failed a build so Ive made this change. AFAIK base-python is outdated and we should move away from it.